contents/handbook/growth/sales/new-hire-onboarding.md

```diff
@@ -0,0 +1,76 @@
+---
+title: New team member onboarding
+sidebar: Handbook
+showTitle: true
+---
+
+## Your first few weeks
+
+Welcome to the PostHog Sales and Customer Success team!  You've done well to make it through our interview process and by 
+doing so we think you'll be a valuable member of the team.  Unlike a lot of companies, we don't have a super-long onboarding 
+process and would prefer you to be up and running with your customer base as quickly as possible.  Here are the things you 
+should focus on in your first few weeks at PostHog to help you achieve that aim.
+
+### Week 1
+
+#### Day 1 Priorities
+
+ - Meet with [Simon](/community/profiles/28895) who will run through this plan and answer any questions you may have.  In addition, come equipped to talk about any nuances around how you prefer to work (e.g. schedules, family time etc.).
+ - Meet with [Mine](/community/profiles/29862) who will get you set up with core tooling:
+   - [Salesforce](https://posthog.lightning.force.com/)
+   - [Vitally](https://posthog.vitally-eu.io/)
+   - [Zendesk](https://posthoghelp.zendesk.com/agent/dashboard)
+   - PostHog [US](https://us.posthog.com/) and [EU](https://eu.posthog.com/) instances (and our production project on the US)
+   - [Jimminy](https://app.jiminny.eu/dashboard)
+ - If you start on a Monday, join your first PostHog All Hands (at 4.30pm UK/8.30am PT) and be prepared to have a strong opinion on whether pineapple belongs on pizza.
+ - If you start on a Monday, join your fist Sales and CS standup.
```

Comment:
typo: first

Reply:
@posthogseb do you know how to suggest an edit directly here? See attached image. 

<img width="853" alt="Screenshot 2024-12-09 at 10 28 11" src="https://github.com/user-attachments/assets/e985011b-f1a2-4ad8-af45-c50993675231">